keyboards/ramonimbao/herringbone/pro/readme.md

```diff
@@ -0,0 +1,15 @@
+# Herringbone Pro
+
+![Herringbone Pro](https://i.imgur.com/cewklY5l.png)
+
+A through-hole 75% keyboard with a rotary encoder and OLED display!
+
+* Keyboard Maintainer: [Ramon Imbao](https://github.com/ramonimbao)
+* Hardware Supported: ATmega32A
+* Hardware Availability: [Herringbone](https://github.com/ramonimbao/Herringbone)
+
+Make example for this keyboard (after setting up your build environment):
+
+    make ramonimbao/herringbone/pro:default
+
```

Comment:
Can you add "clear instructions on how to reset the board into bootloader mode" as per [PR Checklist](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs)?